releasing/do-release.sh

```diff
@@ -40,12 +40,15 @@ $PREFIX git checkout go.mod go.sum
 # if there are no valid current credentials.
 $PREFIX docker login
 echo "$VERSION" > VERSION
-$PREFIX docker build -t "fullstorydev/grpcurl:${VERSION}" .
-rm VERSION
+
+# Enable qemu and binfmt support
+$PREFIX docker run --privileged --rm tonistiigi/binfmt:qemu-v6.1.0 --install all
```

Comment:
What is this for? Does this install something on the host that is needed for the `buildx` stuff below? If so, maybe elaborate in the comment above.

Reply:
Updated the comment, Also had missed earlier to mention about buildx added in docker 19.03 onward. Updated the same.